### PR TITLE
2023 06 fvk incorporate ztl

### PR DESCRIPTION
**Pull request by fvankrieken**
Continuing #5 and #4 for ztl

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -25,17 +25,7 @@
...
-				"python.formatting.blackPath": "/usr/local/py-utils/bin/black",
-				"python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
-				"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
-				"python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
-				"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
-				"python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
-				"python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
-				"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
-				"python.testing.pytestPath": "/usr/local/py-utils/bin/pytest"
+				"python.formatting.blackPath": "/usr/local/py-utils/bin/black"
```
Comment (fvankrieken) [inline .devcontainer/devcontainer.json:28]: Cleaning out detritus here

File: .devcontainer/docker-compose.yml [hunk 2]
```diff
@@ -5,11 +5,6 @@ services:
     build:
       context: ..
       dockerfile: .devcontainer/Dockerfile
-      args:
-        USER_UID: 1000
-        USER_GID: 1000
-    environment:
-       BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
```
Comment (fvankrieken) [inline .devcontainer/docker-compose.yml:12]: User ids set in dockerfile, which from some googling matches some microsoft dev container dockerfile template. Figure I'll leave there

Comment (fvankrieken) [reply .devcontainer/docker-compose.yml:12]: Build engine will continue to handle like we do the rest of .env stuff

**Comment by fvankrieken** [comment]
Running workflow here - https://github.com/NYCPlanning/data-engineering/actions/runs/5223896631

But other than that seems set!

**Comment by fvankrieken** [comment]
Ready to go - cleaned up EDM_DATA which has some rows in the 3 qaqc tables generated with odd format during my prefect testing. I don't want to actually run the action again because I don't want to upload - might be worth adding some options if we actually want to, but for now I think it's good to review/merge

File: db-pluto/pluto_build/06_export.sh [hunk 3]
```diff
@@ -72,4 +72,6 @@ cd ..
 
 wait
 upload "db-pluto" "${VERSION}/${DATE}" &
-upload "db-pluto" "${branchname}/${DATE}"
+upload "db-pluto" "${VERSION}/latest" &
+upload "db-pluto" "${branchname}/${DATE}" &
+upload "db-pluto" "${branchname}/latest"
```
Comment (fvankrieken) [inline db-pluto/pluto_build/06_export.sh:77]: just a missed folder format thing missed with pluto, wasn't exporting to latest

**Review by damonmcc — APPROVED** [review]

**Pull request merged**